### PR TITLE
Fixing broken build for jazz ui

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

Fix broken builds when changes are made to jazz ui repository

### Description of the Change

Added jazz ui as part of the service catalog which enables automated build whenever changes are made to jazz ui repository

### Benefits

Developers can easily make changes to jazz ui and see the changes in dedicated dev environment automatically!

### CODE REVIEW

File: builds/jenkins-build-pack-sls-app/Jenkinsfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ import groovy.transform.Field
 @Field def environment_logical_id
 
 node() {
-
+  try {
   sh 'rm -rf *'
```
Comment (devsatishm) [inline builds/jenkins-build-pack-sls-app/Jenkinsfile]: Now that we are removing the working directory, this line can be removed.

File: core/jazz_environment-event-handler/index.js [hunk 2]
```diff
@@ -571,7 +565,7 @@ function triggerBuildJob(payload, serviceDetails, configData) {
     var type;
     if (payload.service === 'ui' && payload.domain === 'jazz') {
       type = 'ui';
-      buildQuery = `/build?token=${configData.JOB_TOKEN}`;
+      buildQuery = `/buildWithParameters?token=${configData.JOB_TOKEN}&scm_branch=${payload.physical_id}`;
```
Comment (devsatishm) [inline core/jazz_environment-event-handler/index.js]: Did Jenkins interface change from `build` to `buildWithParameters`?

Comment (SiniWilson) [reply core/jazz_environment-event-handler/index.js]: Yes, we need to change build to buildWithParameters as we need to pass the branch name.

File: builds/jenkins-build-pack-sls-app/Jenkinsfile (hunk 1 above)
Comment (SiniWilson) [reply builds/jenkins-build-pack-sls-app/Jenkinsfile]: Removed

**Pull request merged**